.github/workflows/build_msquic_openssl.yml

```diff
@@ -0,0 +1,34 @@
+name: "Build msquic with openssl interop container"
+
+on:
+  schedule:
+    - cron:  '40 01 * * *'
+  workflow_dispatch:
+
+jobs:
+  update_quay_container:
+    if: github.repository == 'openssl/openssl'
+    runs-on: ubuntu-latest
+    steps:
+      - name: Clone microsoft/msquic
+        uses: actions/checkout@v4
+        with:
+          repository: microsoft/msquic
+          ref: main
+          fetch-depth: 0
+          submodules: recursive
+      - name: Login to Quay.io
+        uses: docker/login-action@v3
```

Comment:
Don't bring new dependencies if it's only really needed. https://github.com/openssl/openssl/blob/master/.github/workflows/build_quic_interop_container.yml uses regular docker commands. Just copy and paste it.